.github/workflows/gh-ci.yaml

```diff
@@ -227,7 +227,7 @@ jobs:
 
     - name: build_docs
       run: |
-        cd package && python setup.py build_sphinx -E
+        cd package && python setup.py build_sphinx -E --keep-going
```

Comment:
Are we keeping "--keep-going"?

Reply:
I think this is what gave all warnings rather than just the first, which was very useful, so yes?

Reply:
Agreed, it's no cost to us but will be super useful in future reviews.